### PR TITLE
LPS-148589 Add Relationship label instead of Definition label in Relationship field

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-148589

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 1]
```diff
@@ -306,8 +306,8 @@ public ObjectRelationship updateObjectRelationship(
 	}
 
 	private ObjectField _addObjectField(
-			User user, String name, long objectDefinitionId1,
-			long objectDefinitionId2, String type)
+			Map<Locale, String> labelMap, User user, String name,
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:309]: @danvb112 can you sort and put labelMap after user? 

Reason: We use to put audit fields (like companyId, user, etc) first

Thx :D

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Review by gabrielwas — APPROVED** [review]

**Comment by gabrielwas** [comment]
LGTM :D

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**